hugegraph-test/src/main/java/com/baidu/hugegraph/core/VertexCoreTest.java

```diff
@@ -408,6 +409,97 @@ public void testAddVertexWithInvalidPropertyList() {
         });
     }
 
+    @Test
+    public void testRemoveLeftRangeIndex(){
+        HugeGraph graph = graph();
+        SchemaManager schema = graph.schema();
+
+        schema.propertyKey("updateTime").asLong().create();
+
+        schema.vertexLabel("soft").properties("name", "updateTime")
+              .primaryKeys("name").create();
+
+        schema.indexLabel("softByUpdateTime").onV("soft").range()
+              .by("updateTime").create();
+
+        final long testDataCount = 10L;
+
+        for (int i = 1; i <= testDataCount; i++) {
+            graph.addVertex(T.label, "soft", "name", "soft" + i,
+                            "updateTime", i);
+        }
+        graph.tx().commit();
+
+        long count = graph.traversal().V()
+                          .has("soft", "updateTime",
+                               ConditionP.gt(0))
+                          .limit(-1)
+                          .count()
+                          .next();
+        Assert.assertEquals(testDataCount, count);
+
+        for (int i = 1; i <= testDataCount; i++) {
+            graph.addVertex(T.label, "soft", "name", "soft" + i,
+                            "updateTime", 2 * i);
+        }
+        graph.tx().commit();
+
+        count = graph.traversal().V()
+                     .has("soft", "updateTime",
+                          ConditionP.gt(0))
+                     .limit(-1)
+                     .count()
+                     .next();
+        Assert.assertEquals(testDataCount, count);
+    }
+
```

Comment:
add one test:
vertex label 'person' has two property 'age' and 'score' with range index, and construct left index for both. Then query by 'age' and 'score' by range.